frontend/public/components/_quota.scss

```diff
@@ -5,3 +5,65 @@
 .co-resource-quota-empty {
   font-size: 15px;
 }
+
+.co-resource-quota-chart-row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+  .co-resource-quota-chart {
+    min-width: 150px;
+  }
+}
+
+.co-resource-quota-scope__label {
+  font-weight: 500;
+}
+.co-resource-quota-scope__description {
+  font-size: 12px;
```

Comment:
Here and below WRT font-size:  ideally this would use a var (e.g., $font-size-base - 1) so that if $font-size-base changes, this changes, too.